docker-build-image.sh

```diff
@@ -13,23 +13,30 @@ echoYellow "|    Building the Docker image for development env       |"
 echoYellow "|--------------------------------------------------------|\n"
 
 IMAGE_NAME="kursinfo-web-image"
+DOCKERFILE="Dockerfile-dev"
 
 
 if [ "$ENV" == "dev" ]; then
 
   echo
-  echoYellow "  1. Stop previous Docker image: a name tag is $IMAGE_NAME\n"
+  echoYellow "  1. Creating Dockerfile-dev for development environment\n"
```

Comment:
Do you need to create this file in the script? You have checked in is as well, so it will already be there.

Reply:
You are right and I am agree with you, I made the required changes 🙌